src/proj_internal.h

```diff
@@ -677,6 +677,31 @@ struct projNetworkCallbacksAndData
     void* user_data = nullptr;
 };
 
+struct projGridChunkCache
+{
+    bool enabled = true;
+    std::string filename{};
+    long long max_size = 100 * 1024 * 1024;
```

Comment:
We had previously discussed a cache size of 300mb...

Reply:
indeed. Was changed in proj.ini & doc, but not here. Fixed